src/fu-main.c

```diff
@@ -423,6 +424,18 @@ fu_main_authorize_install_queue (FuMainAuthHelper *helper_ref)
 		return;
 	}
 
+	/* notify the plugins about the transaction */
```

Comment:
I think you'll need similar code in fu-tool right?

Reply:
Urgh, I forgot about that. Maybe move it all to pk-engine.c and just use that directly. Sleep calls. :)

Reply:
Fixed.